index-api/src/Ergvein/Index/API/Models.hs

```diff
@@ -0,0 +1,15 @@
+module Ergvein.Index.API.Models where
+
+import Data.Word
+import Data.Text
+import Ergvein.Aeson
+
+type LockingScriptHash = Text
```

Comment:
Use more standard name PubKeyScriptHash. Add doc comment how the script is hashed and why we use it instead of payment addresses.